api/Db Scripts/CreateNotesTable.sql

```diff
@@ -1,9 +1,31 @@
-use LMPlatform
-create table Notes (
-    Id int primary key identity not null,
-    Text nvarchar(max),
-    Date date not null,
-    Time time not null,
-    UserId int foreign key references Users(UserId),
-    SubjectId int foreign key references Subjects(Id),
-)
+USE educatsb_LMPlatform_FITR
```

Comment:
we should not change already applied scripts. To do this, a new script is created and table is modified